### ISSUE TITLE
add an easily editable example zone using the layout from test zone act 1 form classic sonic simulator

### ISSUE THREAD

**Issue by rubiidev18alt** [RetroKoH/S1Fixed#36]
^

**Comment by RetroKoH** [comment]
I have never played Classic Sonic Simulator. I assume the purpose of this would be for a zone making tutorial, yes?

**Comment by rubiidev18alt** [comment]
> I have never played Classic Sonic Simulator. I assume the purpose of this would be for a zone making tutorial, yes?

yeah basically! also, it's a fun roblox game! you should try it! you can even make your own zones!

**Comment by RetroKoH** [comment]
> > I have never played Classic Sonic Simulator. I assume the purpose of this would be for a zone making tutorial, yes?
> 
> yeah basically! also, it's a fun roblox game! you should try it! you can even make your own zones!

I gotta ask one of my students when I return to work after the holiday. All he does is play it (as opposed to... y'know... studying😅. It'll make for good speaking practice.

**Comment by Rubiidev-IPad** [comment]
> > > I have never played Classic Sonic Simulator. I assume the purpose of this would be for a zone making tutorial, yes?
> > 
> > 
> > yeah basically! also, it's a fun roblox game! you should try it! you can even make your own zones!
> 
> I gotta ask one of my students when I return to work after the holiday. All he does is play it (as opposed to... y'know... studying😅. It'll make for good speaking practice.

alt here! btw, does he have a yt channel?

**Comment by RetroKoH** [comment]
He doesn't. He's only in middle school. I teach mostly primary and lower secondary students

**Comment by RetroKoH** [comment]
I just looked up video of it. It uses the tiles from Stealth's old engine.

I will make a test zone for this purpose but it wont be 1:1 with this C.S.S. level. I will backport a few objects for it though.
